bedrock/cms/management/commands/scrub_exported_cms_data.py

```diff
@@ -0,0 +1,46 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from django.contrib.auth.models import User
+from django.contrib.sessions.models import Session
+from django.core.management.base import BaseCommand
+
+from wagtail.models import Revision
+from wagtail_localize.models import TranslationSource
+
+from bedrock.utils.management.decorators import alert_sentry_on_exception
+
+
+@alert_sentry_on_exception
+class Command(BaseCommand):
+    """Purge Wagtail-managed data that we don't need in the published site
+    which may be either confusing, outdated or too early to publicly share (as a draft)
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument("-q", "--quiet", action="store_true", dest="quiet", default=False, help="If no error occurs, swallow all output.")
+
+    def output(self, *args):
+        if not self.quiet:
+            print(*args)
+
+    def handle(self, *args, **options):
+        self.quiet = options["quiet"]
+
+        user_results = User.objects.all().delete()
+        self.output("Deleted Users:", user_results)
+
+        session_results = Session.objects.all().delete()
+        self.output("Deleted Sessions:", session_results)
+
+        # Delete all Revisions so that the exported DB doesn't contain
+        # pre-publishing versions, stale versions or interim draft versions of
+        # data that may be not yet ready for the public, or now incorrect
+        revisions_result = Revision.objects.all().delete()
+        self.output("Deleted non-live Revisions:", revisions_result)
```

Comment:
so this deletes all non-published content? or would that require another query?

Reply:
although non-published content will in-theory be available on the admin and possibly dev instances.... so maybe it's no big deal? maybe we do need a way to specify non-published content that should only be viewable by a logged in user on the admin instance?

Reply:
> so this deletes all non-published content? 

Yes, that's right. 

Draft versions of pages are saved as Revisions, with each save generating a _new_ Revision. When a draft is published, the content is copied over to the actual relevant Page model's fields. 

This means that Revisions contain all past iterations of a page, plus any unpublished future iteration. Both of these might contain sensitive info, so it's better to just zap them all, and only have the current, live and approved state of the page in the DB that we export. 

In special situations we might want to download a version of the prod DB with revisions, but we can make a separate tool (or add a special flag to this one)

Reply:
>  maybe we do need a way to specify non-published content that should only be viewable by a logged in user on the admin instance?

In terms of viewing on the web, that's how Wagtail works anyway.

In terms of dev/stage DBs containing non-published-and-possibly-sensitive-data, i think we should just scrub the Dev and Stage sqlite DBs at generation, too